src/convertToObject.js

```diff
@@ -3,16 +3,39 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an example in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  let cssArray = sourceString.replace(/(\r\n|\n|\r)/gm, '').split(';');
+
+  for (let i = 0; i < cssArray.length; i++) {
+    cssArray[i] = cssArray[i].trim();
+  };
+
+  cssArray = cssArray
+    .map(el => el.split(':'))
+    .filter(el => el.includes('') !== true);
+
+  cssArray.forEach((el) => {
+    el[0] = el[0].trim();
+    el[1] = el[1].trim();
+  });
+
+  const cssObj = {};
+
+  cssArray.map((el) => {
+    cssObj[el[0]] = el[1];
+  });
```

Comment:
do it in 1 loop, try to use [array destructuring](https://www.freecodecamp.org/news/array-destructuring-in-es6-30e398f21d10/) to avoid using things like el[0]